packages/@aws-cdk/aws-ecs/test/test.generic-log-driver.ts

```diff
@@ -0,0 +1,69 @@
+import { expect, haveResourceLike } from '@aws-cdk/assert';
+import cdk = require('@aws-cdk/core');
+import { Test } from 'nodeunit';
+import ecs = require('../lib');
+
+let stack: cdk.Stack;
+let td: ecs.TaskDefinition;
+const image = ecs.ContainerImage.fromRegistry('test-image');
+
+export = {
+  'setUp'(cb: () => void) {
+    stack = new cdk.Stack();
+    td = new ecs.FargateTaskDefinition(stack, 'TaskDefinition');
+
+    cb();
+  },
+
+  'create a journald log driver with options'(test: Test) {
+    // WHEN
+    td.addContainer('Container', {
+      image,
+      logging: new ecs.GenericLogDriver({
+        logDriver: 'journald',
+        options: {
+          tag: 'hello'
+        }
+      })
+    });
+
+    // THEN
+
+    expect(stack).to(haveResourceLike('AWS::ECS::TaskDefinition', {
+      ContainerDefinitions: [
+        {
+          LogConfiguration: {
+            LogDriver: 'journald',
```

Comment:
Could you see your way to adding top-level classes for these as well, perhaps?

Reply:
At least the journald one, since that's the one you seem to care about the most.